ppml/services/check-service.sh

```diff
@@ -1,6 +1,34 @@
 #!/bin/bash
 # Acceptable arguments: attestation, bigdl-kms, ehsm, kms-utils, pccs, all
 
+sgx() {
+    echo "Detecting SGX..."
+    if [ -e "/dev/sgx" ] && [ -e "/dev/sgx_enclave" ] && [ -e "/dev/sgx_provision" ]; then
+        echo "SGX driver is installed."
+    else
+        echo "SGX driver isn't installed."
+    fi
+
+    echo "Detecting aesmd..."
+    if [ -e "/var/run/aesmd/aesm.socket" ]; then
```

Comment:
Maybe check status of aesmd system service and grep whether `Active: active (running)` is better than brutally `ls`

Reply:
good idea